diff --git a/PKGBUILD b/PKGBUILD
index f235332..e37d155 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -14,11 +14,6 @@ sha512sums=('b35238fa96f7e3201f7c9a70172ecd2c6b0be66e4735ff96c17ae2167f670a67c85
 
 prepare() {
     echo -e '\033[0;31mWARNING : \033[1;33mThis package is deprecated, please use systemd-binfmt instead.\033[0m'
-    read -p "Do you really want to install it anyway ? " yn 
-    case $yn in
-	[Nn]* ) exit;;
-    esac
-    set -e
     cd "${srcdir}/$pkgname-$_pkgver"
     ./bootstrap
 }
