diff --git a/PKGBUILD b/PKGBUILD
index 6b77c77..fb74ed7 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -51,16 +51,6 @@ pkgname=('intel-common-libs'
          'intel-vtune-amplifier'
          'intel-inspector')
 
-if [ -z "$FORCE_PKGEXT" ]; then
-  PKGEXT='.pkg.tar.lzo'
-  msg "Overriding the default PKGEXT with '${PKGEXT}' to save compression time"
-  msg "(change PKGEXT in the PKGBUILD or use FORCE_PKGEXT on the command line"
-  msg " to override PKGEXT set by the package)."
-else
-  msg "Overriding PKGEXT with FORCE_PKGEXT='${FORCE_PKGEXT}'"
-  PKGEXT=$FORCE_PKGEXT
-fi
-
 if [ "$PKGEXT" = ".pkg.tar.xz" ]; then
   plain   "--------------------------------------------------------------------"
   warning "This PKGBUILD may need up to 20 minutes when XZ compressor is used!"
