diff --git a/.SRCINFO b/.SRCINFO
index 043712a..5e564b6 100644
--- a/.SRCINFO
+++ b/.SRCINFO
@@ -7,6 +7,7 @@ pkgbase = lib32-libkate
 	license = BSD
 	makedepends = pkg-config
 	makedepends = git
+	makedepends = python2
 	depends = lib32-libogg
 	depends = lib32-libpng
 	depends = libkate
diff --git a/PKGBUILD b/PKGBUILD
index 9bd11b9..0818e97 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -10,7 +10,7 @@ url="https://wiki.xiph.org/OggKate"
 license=('BSD')
 arch=('x86_64')
 depends=('lib32-libogg' 'lib32-libpng' 'libkate')
-makedepends=('pkg-config' 'git')
+makedepends=('pkg-config' 'git' 'python2')
 source=("https://download.videolan.org/contrib/kate/$_basename-$pkgver.tar.gz"
         0001-Fix-automake-warnings.patch)
 sha512sums=('2635237753abc0f6c43fc07bd0ecc57c2f3c893efa6c62ad08de9191045612477302ed45acd61cdfea7447efd654ebbe0ed2d6a2cbf867eb834581d7ee43377b'
