diff --git a/PKGBUILD b/PKGBUILD
index 6f07791..a32906c 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -84,7 +84,7 @@ options=("!strip")
 ## The final package can be very large (especially if documentation is kept) and
 ## compression can be quite slow.  In most cases, the package is installed
 ## straight away and the package need not be kept, so compression is disabled.
-PKGEXT='.pkg.tar'
+#PKGEXT='.pkg.tar'
 
 prepare() {
     warning "Building Mathematica takes more than 20GiB of space for 'makepkg', and another 10GiB for the pkg tarball."
