diff --git a/.SRCINFO b/.SRCINFO
index a6d2859..992f2e8 100644
--- a/.SRCINFO
+++ b/.SRCINFO
@@ -8,8 +8,6 @@ pkgbase = ttf-hack-powerline-git
 	license = custom:SIL OFL, v1.1
 	license = custom:Bitstream License
 	makedepends = git
-	depends = fontconfig
-	depends = xorg-font-utils
 	provides = ttf-hack
 	conflicts = powerline-fonts
 	conflicts = powerline-fonts-git
diff --git a/PKGBUILD b/PKGBUILD
index 5caffa9..1433e76 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -6,7 +6,6 @@ pkgdesc="Hack for Powerline"
 arch=('any')
 url='https://github.com/powerline/fonts'
 license=('custom:SIL OFL, v1.1' 'custom:Bitstream License')
-depends=(fontconfig xorg-font-utils )
 makedepends=('git')
 provides=(ttf-hack)
 conflicts=(powerline-fonts powerline-fonts-git ttf-hack)
