diff --git a/PKGBUILD b/PKGBUILD
index 1214b3a..bb94385 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -28,6 +28,7 @@ prepare() {
     cd "${srcdir}/apache-ant-${pkgver}"
     unset  ANT_HOME
     export ANT_OPTS="-Duser.home=${srcdir}"
+    export JAVA_HOME=/usr/lib/jvm/default
 
     ./bootstrap.sh
     bootstrap/bin/ant -Ddest=optional -f fetch.xml
@@ -36,6 +37,7 @@ prepare() {
 build() {
     cd "${srcdir}/apache-ant-${pkgver}"
     export ANT_HOME="${srcdir}/apache-ant-${pkgver}/bootstrap"
+    export JAVA_HOME=/usr/lib/jvm/default
 
     bootstrap/bin/ant -lib lib/optional dist
 }
