diff --git a/.SRCINFO b/.SRCINFO
index 64e99ef..c429c29 100644
--- a/.SRCINFO
+++ b/.SRCINFO
@@ -8,6 +8,8 @@ pkgbase = binfmt-support
 	license = GPL
 	makedepends = make
 	makedepends = gcc
+	makedepends = git
+	makedepends = libpipeline
 	depends = autoconf
 	depends = automake
 	source = https://git.savannah.gnu.org/cgit/binfmt-support.git/snapshot/binfmt-support-2.2.0.tar.gz
diff --git a/PKGBUILD b/PKGBUILD
index 71be0a0..3a07115 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -8,7 +8,7 @@ arch=('i686' 'x86_64')
 url="https://git.savannah.gnu.org/cgit/binfmt-support.git/"
 license=('GPL')
 depends=('autoconf' 'automake')
-makedepends=('make' 'gcc')
+makedepends=('make' 'gcc' 'git' 'libpipeline')
 source=("https://git.savannah.gnu.org/cgit/binfmt-support.git/snapshot/$pkgname-$_pkgver.tar.gz")
 sha512sums=('b35238fa96f7e3201f7c9a70172ecd2c6b0be66e4735ff96c17ae2167f670a67c856d0fc1b0d50bf3c14f4a35c5e9927a848814791ea9be928dff82426cb0da4')
 
