diff --git a/.SRCINFO b/.SRCINFO
index a3bb870..6bbffd3 100644
--- a/.SRCINFO
+++ b/.SRCINFO
@@ -24,7 +24,7 @@ pkgbase = firefox-beta-bin
 	source = firefox-73.0rc3.tar.bz2::https://ftp.mozilla.org/pub/firefox/candidates/73.0-candidates/build3/linux-x86_64/en-US/firefox-73.0.tar.bz2
 	source = firefox-beta.sh
 	source = firefox-beta-bin.desktop
-	sha256sums = 3dc41b474cce48a172a424794e46690f725911abaf60fca33655aae24d0e224f
+	sha256sums = 96f6a99dc984590acb93814e28e52028f2c764302426bef90c70a7e5a115b037
 	sha256sums = 4bec62032e49c28ff27750abddbdbdbb1a4b5cba719c39498968fe53adee790b
 	sha256sums = 210f13ea47c4b96387f26ee7fc4dfc5c192cfb169aef2a13303fbd1ee58b3761
 
diff --git a/PKGBUILD b/PKGBUILD
index 4b34de2..b87a0fe 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -25,7 +25,7 @@ install=$pkgname.install
 source=("https://ftp.mozilla.org/pub/firefox/releases/$pkgver/linux-x86_64/en-US/firefox-$pkgver.tar.bz2"
         "$_pkgname.sh"
         "$pkgname.desktop")
-sha256sums=('3dc41b474cce48a172a424794e46690f725911abaf60fca33655aae24d0e224f'
+sha256sums=('96f6a99dc984590acb93814e28e52028f2c764302426bef90c70a7e5a115b037'
             '4bec62032e49c28ff27750abddbdbdbb1a4b5cba719c39498968fe53adee790b'
             '210f13ea47c4b96387f26ee7fc4dfc5c192cfb169aef2a13303fbd1ee58b3761')
 # RC
