diff --git a/.SRCINFO b/.SRCINFO
index 82e1964..d2bc6f0 100644
--- a/.SRCINFO
+++ b/.SRCINFO
@@ -8,7 +8,7 @@ pkgbase = i2p
 	install = i2p.install
 	arch = any
 	license = GPL2
-	makedepends = apache-ant
+	makedepends = apache-ant-10
 	makedepends = java-environment>=7
 	makedepends = gettext
 	depends = java-runtime>=7
diff --git a/PKGBUILD b/PKGBUILD
index 671c581..82f44ed 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -16,7 +16,7 @@ url="https://geti2p.net"
 license=('GPL2')
 arch=('any')
 depends=('java-runtime>=7' 'java-service-wrapper')
-makedepends=('apache-ant' 'java-environment>=7')
+makedepends=('apache-ant-10' 'java-environment>=7')
 [[ "$LG2" != 'en' ]] && makedepends+=('gettext')
 #optdepends=('gtk2: for rrd graphs')
 conflicts=('i2p-bin' 'i2p-dev')
