diff --git a/PKGBUILD b/PKGBUILD
index 4f63608..92bf5af 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -11,8 +11,8 @@ pkgdesc="A distributed anonymous network"
 url="https://geti2p.net"
 license=('GPL2')
 arch=('any')
-depends=('java-runtime>=8' 'java-service-wrapper')
-makedepends=('java-environment>=8' 'apache-ant')
+depends=('java-runtime=11' 'java-service-wrapper')
+makedepends=('java-environment=11' 'apache-ant')
 [[ "$LG2" != 'en' ]] && makedepends+=('gettext')
 #optdepends=('gtk2: for rrd graphs')
 conflicts=('i2p-bin' 'i2p-dev')
