diff --git a/.SRCINFO b/.SRCINFO
index 355e2c6..7172649 100644
--- a/.SRCINFO
+++ b/.SRCINFO
@@ -19,7 +19,7 @@ pkgbase = lib32-gst-plugins-ugly
 	depends = lib32-x264
 	depends = lib32-opencore-amr
 	depends = gst-plugins-ugly
-	source = git+https://github.com/freedesktop/gstreamer-gst-plugins-ugly.git#commit=e1bf2aa184f83ff9fd5b7850c460129100ac6d1a
+	source = gst-plugins-ugly::git+https://github.com/freedesktop/gstreamer-gst-plugins-ugly.git#commit=e1bf2aa184f83ff9fd5b7850c460129100ac6d1a
 	source = gst-common::git+https://github.com/freedesktop/gstreamer-common.git
 	sha256sums = SKIP
 	sha256sums = SKIP
diff --git a/PKGBUILD b/PKGBUILD
index f5cf7e9..c7cf835 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -19,7 +19,7 @@ _commit=e1bf2aa184f83ff9fd5b7850c460129100ac6d1a  # tags/1.14.4^0
 #        "gst-common::git+https://anongit.freedesktop.org/git/gstreamer/common")
 # I'm not sure if I'm just unlucky or whatever, the git server from freedesktop is too slow
 # So switch to github mirrors
-source=("git+https://github.com/freedesktop/gstreamer-gst-plugins-ugly.git#commit=$_commit"
+source=("gst-plugins-ugly::git+https://github.com/freedesktop/gstreamer-gst-plugins-ugly.git#commit=$_commit"
         "gst-common::git+https://github.com/freedesktop/gstreamer-common.git")
 
 sha256sums=('SKIP'
