diff --git a/.SRCINFO b/.SRCINFO
index d33d753..53cef3a 100644
--- a/.SRCINFO
+++ b/.SRCINFO
@@ -16,7 +16,9 @@ pkgbase = lib32-libplacebo
 	depends = lib32-libepoxy
 	provides = libplacebo.so
 	source = https://code.videolan.org/videolan/libplacebo/-/archive/v2.43.1/libplacebo-v2.43.1.tar.gz
+	source = vulkan-header.patch::https://code.videolan.org/videolan/libplacebo/-/commit/45e19e7bbbbfceb197d8826c775e16ef536a4565.patch
 	sha512sums = 62463dfee6e6fd450395e42d7af2b6e0caa0b183c0d00f5ba0c5ddc893a47cdd0371d37fc72a7361228324b0dc9c55dd0309a5a7210cf753618fde18a094abcb
+	sha512sums = d0491d81afb27f8ab6adbbf369a32a5c23222e7b2c071fa45f26ae682a3ac1d25248b770398d0793cc1c4ea03d172834bce5b53e5297f8cb929f554bcfc5726f
 
 pkgname = lib32-libplacebo
 
diff --git a/PKGBUILD b/PKGBUILD
index e446deb..2597566 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -23,13 +23,20 @@ makedepends=(
         'vulkan-headers'
         )
 provides=('libplacebo.so')
-source=("https://code.videolan.org/videolan/libplacebo/-/archive/v${pkgver}/${_pkgbasename}-v${pkgver}.tar.gz")
-sha512sums=('62463dfee6e6fd450395e42d7af2b6e0caa0b183c0d00f5ba0c5ddc893a47cdd0371d37fc72a7361228324b0dc9c55dd0309a5a7210cf753618fde18a094abcb')
+source=(
+        "https://code.videolan.org/videolan/libplacebo/-/archive/v${pkgver}/${_pkgbasename}-v${pkgver}.tar.gz"
+        vulkan-header.patch::https://code.videolan.org/videolan/libplacebo/-/commit/45e19e7bbbbfceb197d8826c775e16ef536a4565.patch
+        )
+sha512sums=(
+        '62463dfee6e6fd450395e42d7af2b6e0caa0b183c0d00f5ba0c5ddc893a47cdd0371d37fc72a7361228324b0dc9c55dd0309a5a7210cf753618fde18a094abcb'
+        'd0491d81afb27f8ab6adbbf369a32a5c23222e7b2c071fa45f26ae682a3ac1d25248b770398d0793cc1c4ea03d172834bce5b53e5297f8cb929f554bcfc5726f'
+        )
 
 prepare() {
   cd ${_pkgbasename}-v${pkgver}
 
   # Patching if needed
+  patch -Np1 -i "$srcdir"/vulkan-header.patch
 }
 
 build() {
