diff --git a/.SRCINFO b/.SRCINFO
index 20e6c28..1e59428 100644
--- a/.SRCINFO
+++ b/.SRCINFO
@@ -5,14 +5,14 @@ pkgbase = lib32-zbar
 	url = https://github.com/procxx/zbar
 	arch = x86_64
 	license = LGPL
-	makedepends = lib32-imagemagick
-	makedepends = lib32-libxv
 	makedepends = lib32-python2
-	makedepends = lib32-gtk2
-	makedepends = pygtk
-	makedepends = lib32-v4l-utils
 	makedepends = xmlto
+	makedepends = docbook-xsl
 	depends = zbar
+	depends = lib32-imagemagick
+	depends = lib32-libxv
+	depends = lib32-v4l-utils
+	depends = lib32-dbus
 	source = zbar-0.22.tar.gz::https://github.com/mchehab/zbar/archive/0.22.tar.gz
 	source = imagemagick7.patch
 	sha512sums = bef1078f7fe0c943e2ca7348f45e3bb5744e807c5135051fccf2c4499d52515558e98f61620e2ef2c993f35b8104d886d0639b7f0e6f8979caa5dcc49df7c931
diff --git a/PKGBUILD b/PKGBUILD
index d7527fa..83c8994 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -12,8 +12,8 @@ pkgdesc="Application and library for reading bar codes from various sources. 32b
 arch=('x86_64')
 url="https://github.com/procxx/zbar"
 license=('LGPL')
-depends=('zbar')
-makedepends=('lib32-imagemagick' 'lib32-libxv' 'lib32-python2' 'lib32-gtk2' 'pygtk' 'lib32-v4l-utils' 'xmlto')
+depends=('zbar' 'lib32-imagemagick' 'lib32-libxv' 'lib32-v4l-utils' 'lib32-dbus')
+makedepends=('lib32-python2' 'xmlto' 'docbook-xsl')
 source=("$_pkgname-$pkgver.tar.gz::https://github.com/mchehab/zbar/archive/$pkgver.tar.gz"
         imagemagick7.patch)
 sha512sums=('bef1078f7fe0c943e2ca7348f45e3bb5744e807c5135051fccf2c4499d52515558e98f61620e2ef2c993f35b8104d886d0639b7f0e6f8979caa5dcc49df7c931'
