diff --git a/.SRCINFO b/.SRCINFO
index a2dd475..b355e20 100644
--- a/.SRCINFO
+++ b/.SRCINFO
@@ -9,6 +9,7 @@ pkgbase = lib32-x265
 	makedepends = nasm
 	depends = lib32-gcc-libs
 	depends = lib32-numactl
+	depends = x265
 	provides = libx265.so
 	source = https://bitbucket.org/multicoreware/x265/downloads/x265_3.2.1.tar.gz
 	sha256sums = fb9badcf92364fd3567f8b5aa0e5e952aeea7a39a2b864387cec31e3b58cbbcc
diff --git a/PKGBUILD b/PKGBUILD
index e5807c5..ee8ea5e 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -12,7 +12,7 @@ pkgdesc='Open Source H265/HEVC video encoder. 32bit libraries.'
 arch=('x86_64')
 url='https://bitbucket.org/multicoreware/x265'
 license=('GPL')
-depends=('lib32-gcc-libs'  'lib32-numactl')
+depends=('lib32-gcc-libs'  'lib32-numactl' $_basename)
 makedepends=('cmake' 'nasm')
 provides=('libx265.so')
 source=("https://bitbucket.org/multicoreware/x265/downloads/x265_${pkgver}.tar.gz")
