diff --git a/.SRCINFO b/.SRCINFO
index ea59ca4..89cd544 100644
--- a/.SRCINFO
+++ b/.SRCINFO
@@ -1,18 +1,17 @@
 pkgbase = lib32-lv2
 	pkgdesc = Successor to the LADSPA audio plug-in standard
 	pkgver = 1.16.0
-	pkgrel = 1
+	pkgrel = 2
 	url = http://lv2plug.in/
 	arch = x86_64
 	license = LGPL
 	license = custom
-	makedepends = lib32-python
+	makedepends = python
 	makedepends = lib32-libsndfile
 	makedepends = lib32-gtk2
-	makedepends = lv2
-	optdepends = libsndfile: Example sampler
-	optdepends = gtk2: Example sampler
-	optdepends = python2: Scripts
+	depends = lv2
+	optdepends = lib32-libsndfile: Example sampler
+	optdepends = lib32-gtk2: Example sampler
 	provides = lib32-lv2core
 	conflicts = lib32-lv2core
 	replaces = lib32-lv2core
diff --git a/PKGBUILD b/PKGBUILD
index c3971c1..38c383b 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -4,15 +4,15 @@
 _basename=lv2
 pkgname=lib32-${_basename}
 pkgver=1.16.0
-pkgrel=1
+pkgrel=2
 pkgdesc="Successor to the LADSPA audio plug-in standard"
 url="http://lv2plug.in/"
 license=('LGPL' 'custom')
 arch=('x86_64')
-makedepends=('lib32-python' 'lib32-libsndfile' 'lib32-gtk2' 'lv2')  #lib32-python should be more correct
-optdepends=('libsndfile: Example sampler'
-            'gtk2: Example sampler'
-            'python2: Scripts')
+depends=('lv2')
+makedepends=('python' 'lib32-libsndfile' 'lib32-gtk2')
+optdepends=('lib32-libsndfile: Example sampler'
+            'lib32-gtk2: Example sampler')
 provides=('lib32-lv2core')
 conflicts=('lib32-lv2core')
 replaces=('lib32-lv2core')
@@ -25,33 +25,27 @@ sha512sums=('ead6d590cded5dd7a548d6ffe0f2f9f8efadfa7bb9e8b4fa0aea6664ccdfbb3ca69
 
 build() {
   cd "$srcdir/$_basename-$pkgver"
-  export CC='gcc -m32'
-  export CXX='g++ -m32'
+  export CFLAGS="$CFLAGS -m32"
+  export LDFLAGS="$LDFLAGS -m32"
   export PKG_CONFIG_PATH='/usr/lib32/pkgconfig'
   
-  # let wscript(s) find the custom waf scripts
-  mkdir -pv tools
-  touch __init__.py
-  cp -v waflib/extras/{autowaf,lv2}.py tools/
-  mkdir -pv plugins/tools/
-  cp -v waflib/extras/{autowaf,lv2}.py plugins/tools/
-  rm -rv waflib
-  sed -e 's/waflib.extras/tools/g' \
-      -e "s/load('autowaf'/load('autowaf', tooldir='tools'/g" \
-      -e "s/load('lv2'/load('lv2', tooldir='tools'/g" \
-      -i {,plugins/,plugins/*/}wscript
   patch -Np1 -i "../lv2-1.16.0-fix_lv2_validate.patch"
    # --docs is currently broken: https://gitlab.com/lv2/lv2/issues/28
-  waf -vv configure --prefix=/usr \
+  ./waf -vv configure --prefix=/usr \
                 --libdir=/usr/lib32 \
                 --test
-  waf -vv build $MAKEFLAGS
+  ./waf -vv build $MAKEFLAGS
+}
+
+check() {
+  cd "$srcdir/$_basename-$pkgver"
+  ./waf test
 }
 
 package() {
   cd "$srcdir/$_basename-$pkgver"
 #REMOVE includes and others
-  waf install --destdir="$pkgdir"
+  ./waf install --destdir="$pkgdir"
   rm ${pkgdir}/usr/bin ${pkgdir}/usr/include ${pkgdir}/usr/share -Rf
 }
 
