diff --git a/PKGBUILD b/PKGBUILD
index 55f63ec..725cc0b 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -14,10 +14,12 @@ makedepends=('git' 'cmake' 'dotnet-sdk>=2.0')
 depends=('icu' 'openssl-1.0')
 source=($pkgname::"git+https://github.com/PowerShell/PowerShell.git#tag=v$_pkgver"
         'powershell-native::git+https://github.com/PowerShell/PowerShell-Native.git'
-        'Microsoft.PowerShell.SDK.csproj.TypeCatalog.targets')
+        'Microsoft.PowerShell.SDK.csproj.TypeCatalog.targets'\
+        'libpsl-remove-Werror.patch')
 md5sums=('SKIP'
          'SKIP'
-         '56f02557575a6022b60be609951eee78')
+         '56f02557575a6022b60be609951eee78'
+         'a1bfd21b98d88a072db39a62e734f6e4')
 install=powershell.install
 
 prepare() {
@@ -60,6 +62,7 @@ build() {
 
   ## Build native component
   pushd $srcdir/powershell-native/src/libpsl-native
+  git apply "$srcdir/libpsl-remove-Werror.patch"
   cmake -DCMAKE_BUILD_TYPE=Debug .
   make -j
   popd
diff --git a/libpsl-remove-Werror.patch b/libpsl-remove-Werror.patch
new file mode 100644
index 0000000..f8d361a
--- /dev/null
+++ b/libpsl-remove-Werror.patch
@@ -0,0 +1,13 @@
+diff --git a/src/libpsl-native/CMakeLists.txt b/src/libpsl-native/CMakeLists.txt
+index e4f33cc..19dcb08 100644
+--- a/src/libpsl-native/CMakeLists.txt
++++ b/src/libpsl-native/CMakeLists.txt
+@@ -3,7 +3,7 @@ project(PSL-NATIVE)
+ 
+ # Can't use add_compile_options with 2.8.11
+ set(CMAKE_BUILD_TYPE "Release")
+-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Werror -fstack-protector-strong -fpie -DFORTIFY_SOURCE=2 -O2")
++set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -fstack-protector-strong -fpie -DFORTIFY_SOURCE=2 -O2")
+ 
+ if (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
+     set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,-z,relro,-z,now")
