diff --git a/.SRCINFO b/.SRCINFO
index b42aa6b..7c2d29f 100644
--- a/.SRCINFO
+++ b/.SRCINFO
@@ -9,6 +9,7 @@ pkgbase = ruby-ethon
 	license = MIT
 	depends = ruby
 	depends = ruby-ffi
+	depends = ruby-rdoc
 	noextract = ethon-0.11.0.gem
 	options = !emptydirs
 	source = https://rubygems.org/downloads/ethon-0.11.0.gem
diff --git a/PKGBUILD b/PKGBUILD
index 2ff39f0..935b8f6 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -9,7 +9,7 @@ pkgdesc='Libcurl wrapper.'
 arch=(any)
 url='https://github.com/typhoeus/ethon'
 license=(MIT)
-depends=('ruby' 'ruby-ffi')
+depends=('ruby' 'ruby-ffi' 'ruby-rdoc')
 options=(!emptydirs)
 source=(https://rubygems.org/downloads/$_gemname-$pkgver.gem)
 noextract=($_gemname-$pkgver.gem)
