diff --git a/.SRCINFO b/.SRCINFO
index fdc7cc6..44c4975 100644
--- a/.SRCINFO
+++ b/.SRCINFO
@@ -8,6 +8,7 @@ pkgbase = ruby-net-http-persistent
 	arch = any
 	license = MIT
 	depends = ruby
+	depends = ruby-rdoc
 	noextract = net-http-persistent-3.0.0.gem
 	options = !emptydirs
 	source = https://rubygems.org/downloads/net-http-persistent-3.0.0.gem
diff --git a/PKGBUILD b/PKGBUILD
index 2d00189..fc8a4d0 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -9,7 +9,7 @@ pkgdesc='Manages persistent connections using Net::HTTP plus a speed fix for Rub
 arch=(any)
 url='http://docs.seattlerb.org/net-http-persistent'
 license=(MIT)
-depends=(ruby)
+depends=('ruby' 'ruby-rdoc')
 options=(!emptydirs)
 source=(https://rubygems.org/downloads/$_gemname-$pkgver.gem)
 noextract=($_gemname-$pkgver.gem)
