diff --git a/.SRCINFO b/.SRCINFO
index 0f58ffc..9eb4b13 100644
--- a/.SRCINFO
+++ b/.SRCINFO
@@ -8,7 +8,7 @@ pkgbase = thunderbird-extension-enigmail-git
 	license = GPL3
 	makedepends = git
 	makedepends = zip
-	makedepends = python2
+	makedepends = python
 	makedepends = perl
 	depends = gnupg
 	depends = thunderbird
diff --git a/PKGBUILD b/PKGBUILD
index ec5ecaa..9f3ba12 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -14,7 +14,7 @@ arch=('any')
 url="https://www.enigmail.net/"
 license=('MPL' 'GPL3')
 depends=('gnupg' 'thunderbird')
-makedepends=('git' 'zip' 'python2' 'perl')
+makedepends=('git' 'zip' 'python' 'perl')
 provides=('thunderbird-extension-enigmail' 'thunderbird-enigmail')
 conflicts=('thunderbird-extension-enigmail' 'thunderbird-enigmail')
 source=("enigmail-git::git+https://gitlab.com/enigmail/enigmail.git")
