diff --git a/.SRCINFO b/.SRCINFO
index d42431b..65f1474 100644
--- a/.SRCINFO
+++ b/.SRCINFO
@@ -7,6 +7,7 @@ pkgbase = v2ray-plugin
 	url = https://github.com/shadowsocks/v2ray-plugin
 	arch = x86_64
 	license = MIT
+	makedepends = git
 	makedepends = go
 	conflicts = v2ray-plugin-git
 	conflicts = v2ray-plugin-bin
diff --git a/PKGBUILD b/PKGBUILD
index 78fda3a..4cd3d2f 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -7,7 +7,7 @@ pkgdesc='Yet another SIP003 plugin for shadowsocks, based on v2ray'
 arch=('x86_64')
 url='https://github.com/shadowsocks/v2ray-plugin'
 license=('MIT')
-makedepends=('go')
+makedepends=(git go)
 conflicts=("${pkgname}-git" "${pkgname}-bin")
 source=($pkgname::"git+https://github.com/shadowsocks/$pkgname.git#tag=v$pkgver"
         "https://raw.githubusercontent.com/shadowsocks/$pkgname/master/LICENSE")
