diff --git a/PKGBUILD b/PKGBUILD
index e6e49f1..6033ee5 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -12,8 +12,6 @@ url='https://github.com/DeadSix27/waifu2x-converter-cpp'
 license=('MIT')
 depends=('opencv'
          'cuda'
-         'gtk3'
-         'hdf5'
          )
 makedepends=('cmake'
              'git'
@@ -30,9 +28,8 @@ pkgver() {
   echo $(git describe --long --tags | tr - .)
 }
 
-prepare() {
-  mkdir -p build
-
+build() {
+  mkdir build
   cd build
   cmake ../waifu2x-converter-cpp \
     -DCMAKE_BUILD_TYPE=None \
@@ -42,10 +39,7 @@ prepare() {
     -DCUDA_HOST_COMPILER=/opt/cuda/bin/gcc \
     -DINSTALL_MODELS=ON \
     -DOPENCV_PREFIX=/usr
-}
-
-build() {
-  make -C build
+  make
 }
 
 package() {
