diff --git a/.SRCINFO b/.SRCINFO
index 767d7d1..4bc9f0f 100644
--- a/.SRCINFO
+++ b/.SRCINFO
@@ -8,7 +8,7 @@ pkgbase = waifu2x-converter-cpp-cuda-git
 	makedepends = cmake
 	makedepends = git
 	makedepends = opencl-headers
-	depends = opencv-cuda
+	depends = opencv
 	depends = opencl-icd-loader
 	provides = waifu2x-converter-cpp
 	conflicts = waifu2x-converter-cpp
diff --git a/PKGBUILD b/PKGBUILD
index 4b287dc..eaf7f30 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -10,7 +10,7 @@ pkgdesc="Image Super-Resolution for Anime-Style-Art. (re-implementation in C++ u
 arch=('x86_64')
 url='https://github.com/DeadSix27/waifu2x-converter-cpp'
 license=('MIT')
-depends=('opencv-cuda'
+depends=('opencv'
          'opencl-icd-loader'
          )
 makedepends=('cmake'
