diff --git a/.SRCINFO b/.SRCINFO
index 2b16255..572f9fc 100644
--- a/.SRCINFO
+++ b/.SRCINFO
@@ -10,7 +10,7 @@ pkgbase = waifu2x-converter-cpp-cuda-git
 	makedepends = cmake
 	makedepends = git
 	makedepends = opencl-headers
-	makedepends = gcc7
+	makedepends = ocl-icd
 	depends = opencv
 	depends = cuda
 	provides = waifu2x-converter-cpp
diff --git a/PKGBUILD b/PKGBUILD
index ae85f17..d552fce 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -16,7 +16,7 @@ depends=('opencv'
 makedepends=('cmake'
              'git'
              'opencl-headers'
-             'gcc7'
+             'ocl-icd'
              )
 provides=('waifu2x-converter-cpp')
 conflicts=('waifu2x-converter-cpp')
