diff --git a/PKGBUILD b/PKGBUILD
index cee6dfd..a7abda9 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -13,7 +13,7 @@ conflicts=()
 replaces=()
 backup=()
 md5sums=('c5f7dc52c96e606e403751a4c482d785')
-source=('https://web.archive.org/web/20150802061047/http://support.wdc.com/download/archive/pciscsi.exe')
+source=('https://raw.githubusercontent.com/FadeMind/archpkgbuilds/master/AUR/customized/wd719x-firmware/pciscsi.exe')
 noextract=('pciscsi.exe')
 
 build() {
